lib/tasks/move_results_to_attempt_tables.rake

```diff
@@ -3,7 +3,43 @@
 namespace :results do
   desc "Migrates all results to attempts"
   task migrate_attempts: [:environment] do
-    Result.find_each(&:create_or_update_attempts)
+    ActiveRecord::Base.connection.execute <<-SQL.squish
+    INSERT IGNORE INTO result_attempts (value, attempt_number, result_id)
```

Comment:
What does the `IGNORE` part do here?

Reply:
another option would be to do another query to figure out which result_ids already have attempts, but that would make it a good more complicated